test/integration/grid_data_sets.py

```diff
@@ -0,0 +1,140 @@
+"""
+Test data sets for DmapWrite
+"""
+from pydarn import DmapScalar, DmapArray
+from collections import OrderedDict
+from numpy import array, int16, int32, inf, float32
+
```

Comment:
A bit confusing to import array on its own, since array is used as a variable name in other places in the code.  It would be easier for me to read if you just did the standard `import numpy as np`

Reply:
Fixed with the other files as well.